### PR TITLE
Feature/contact form

### PR DESCRIPTION
**Pull request by flibbertigibbet**
JotForms conflicts with jQuery, which is why I've put the little bit of validation logic for the form into plain Javascript, inline.

### CODE REVIEW

**Comment by flibbertigibbet** [comment]
image

image

**Comment by flibbertigibbet** [comment]
Closes #395.

File: python/cac_tripplanner/templates/contact-form-mixin.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                    <label>
+                        <input type="checkbox" id="emailAgree" value=""
+                            name="agree"> I agree to receive emails from GoPhillyGo </label>
+                </div>
+                <br />
+                <input type="submit" id="emailSubscribe" value="subscribe" disabled />
+            </div>
+            <input type="hidden" id="simple_spc" name="simple_spc" value="51094584186159" />
+            <script type="text/javascript">
+                document.getElementById("si" + "mple" + "_spc").value = "51094584186159-51094584186159";
```
Comment (kshepard) [inline python/cac_tripplanner/templates/contact-form-mixin.html]: Any idea what's up with this? Some kind of security through obscurity?

**Comment by kshepard** [comment]
+1

File: python/cac_tripplanner/templates/contact-form-mixin.html (hunk 1 above)
Comment (flibbertigibbet) [inline python/cac_tripplanner/templates/contact-form-mixin.html]: It's a mystery.  That's a copy-paste from the provided snippet.

**Pull request merged**